tests/integration/values.yaml

```diff
@@ -0,0 +1,149 @@
+# configure Connaisseur deployment
```

Comment:
Do you really want to maintain a copy of the values.yml? I think using a copy/the default one and make targeted changes is a) easier to maintain and b) makes sure we actually test the current state and not some old things

Reply:
okokok